test/fluree/json_ld/impl/expand_test.cljc

```diff
@@ -440,6 +440,80 @@
             "http://example.com/ns#favColor" [{:id "http://example.com/ns#red", :idx [:ex/favColor 0]}
                                               {:id "http://example.com/ns#green", :idx [:ex/favColor 1]}]}))))
 
+(deftest shacl-embedded-nodes
+    (testing "clojure kws"
```

Comment:
Super minor nitpick, but should all of the `(testing ...)` forms be unindented a couple spaces?

Reply:
Whoops, I somehow missed this comment and merged when I saw the approval. I think you're right, which will unfortunately be annoying to whoever touches this file next 😅 My apologies to that future json-ld lib developer!

Reply:
Yeah sorry I meant to mention it in the approval comment. No big deal.